### ISSUE TITLE
Do not force specific SDK version 3.1.409

### ISSUE THREAD

**Issue by prosconi** [Dzoukr/Dapper.FSharp#33]
Self explanatory

https://github.com/Dzoukr/Dapper.FSharp/blob/e864e07d8be727633a761e3caf16b61e554956eb/global.json#L1-L5

**Comment by Dzoukr** [comment]
What's the problem with it? It's for local development and can be deleted if you don't like it.

**Comment by prosconi** [comment]
This file is in source control so the process for people who do not have that specific SDK would be to:

**either**

* git pull
* delete or modify global.json
* make sure you don't check it in

or **install that specific SDK version**

**Comment by Dzoukr** [comment]
> or install that specific SDK version

Yup. Exactly. And this is how it should be. `global.json` is like a `paket.lock` - a well-known definition of expected dependencies for the correct (repeatable) build. From a CI perspective, it's a must-have thing (you don't want the local build environment to affect the build output). There can be so many dragons between SDK versions (like push to Nuget feed) and if you want to be safe, use the locked version (https://docs.microsoft.com/en-us/dotnet/core/tools/global-json).

In other words: The project is responsible for setting expectations. If you want to contribute, you can:

- follow expected SDK
- or use the latest one locally (by changing/deleting `global.json`) and do the PR with the hope it will be fine in other SDK version

But the final version will be built/pushed using locked SDK anyway. 🤷‍♀️
